README.md

```diff
@@ -87,6 +87,7 @@ The config file is a yaml formatted file with the following fields:
 | `cooldown_duration` | No               | 86400 (24 hours)                               | Yes                          | time in seconds for a discord user to be offline before it's puppet disconnects from irc                                                                                 |
 | `show_joinquit`     | No               | false                                          | yes                          | displays JOIN, PART, QUIT, KICK on discord                                                                                                                               |
 | `max_nick_length`   | No               | 30                                             | yes                          | Maximum allowed nick length                                                                                                                                              |
+| `connection_limit`  | Yes              |                                                | Yes                          | How many connections to IRC (including our listener) to spawn (limit of 0 or less means unlimited)                                                                       |
```

Comment:
This should default to 0

Reply:
The default is 0 (that's the default value for an integer) but I guess it needs to be documented in `README.md`? I'll submit that change for `README.md` in a moment.

Reply:
Ah, cool, didn't realise the zero value was automatically used. Neat!